OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/NewCourseOutlineAdapter.java

```diff
@@ -406,6 +406,12 @@ public void onResult(DownloadEntry.DownloadedState state) {
                                         new View.OnClickListener() {
                                             @Override
                                             public void onClick(View v) {
+                                                /**
```

Comment:
This isn't a javadoc rather a devnote. So we should use `/*` in this case instead of `/**`

Reply:
Redirections for e.g. `{@link DownloadEntry#url}` only work in /** , java doc.

Reply:
ok, it makes sense to ignore my nit then.